vendor/engines/secure_rooms/README.md

```diff
@@ -67,6 +56,19 @@ The door security hardware communicates through an API with the following endpoi
 
       ```
       { response:  "select_account",
-        tablet_id: <integer id>,
-        accounts:  [<list of account data TBD>] }
+        tablet_identifier: <id of associated tablet>,
+        name: "Full Nameofuser",
+        accounts:  [<list of all account model attributes>] }
+      ```
+
+  In addition, sending identifiers that do not map to existing records will generate a not found:
+
+  **Not Found:** One of the input identifiers does not match an existing record.
+
+  * **Code:** 404 Not Found
```

Comment:
Nice. I've coded `200` as `grant` and `300` as `select_account`; I'm treating _anything else_ as a deny. For a `403` I just straight `deny, and anything else (e.g. 404, 400, 500) I log a error and then deny.

Do you think instead of logging an error, though, I should report to rollbar? I think maybe yes...